stripe-ruby-mock.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'stripe', '~> 1.31'
+  gem.add_dependency 'stripe', '~> 2.8'
```

Comment:
I've ran test suite against `2.0`, but it had some specs failed, and it worked fine with `2.1` only. I guess there were some fixes in 2.1. 
There's not such a big difference between 2.0 and 2.1, so I think it will worth upgrading to 2.1 or even to the most recent version, but 2.1 should work since tests don't fail.